README.md

```diff
@@ -111,12 +111,16 @@ INSERT INTO person (id, first_name, last_name) VALUES ($1, $2, $3)
 Parameters are referenced using the same identifiers when binding these:
 
 ```java
-connection
```

Comment:
I wouldn't change this directly (certainly not to a user of `subscribe()` which is almost never correct).  I think the key either needs to be solely the former name change, or _possibly_ a more complex example with a `flatMap()`.  I lean towards keeping this simpler (eliminating the outer layer where you get the connection from the `Mono`, but I can see an argument where it's much more representative of real usage.

Reply:
OK, I will try to provide a more complex example

Reply:
@nbenjamin 
Could you review it now , I think that it's much better to show how to get the connection from the Mono . I have used **map** function . What do you think about it ?

Reply:
Please remove `.block().blockFirst();` as calling these methods in a reactive context is considered an antipattern.

Reply:
@mp911de  I did it , could you review it now ?